### PR TITLE
Operation: Reduce Roundstart Waiting

### PR DESCRIPTION
**Pull request by Tupinambis**
## About The Pull Request

This PR gives marines basic uniform kits and full hunger roundstart, and reduces the delay for the dropship by 10 minutes.

Xenos will also now always be mature upon evolution while the shuttle is unable to be launched.

## Why It's Good For The Game

Nobody likes waiting forever for the actual fun part of the round to start, so these changes should help.

## Changelog
:cl:
tweak: marines now start with full hunger and their basic uniform set on spawn.
tweak: Dropship roundstart delay before it can head towards the LZ has been reduced by 10 minutes.
balance: Xenomorphs now evolve into a mature form while the Alamo is unable to be launched.
/:cl:

### CODE REVIEW

**Comment by Kuro-ai** [comment]
nice,I like it

**Comment by Spyroshark** [comment]
we might want to make xenos start at mature or something, since marines will be arriving faster

**Comment by Pariah919** [comment]
>nerfing survivors even more
sad

File: code/datums/gamemodes/distress.dm [hunk 1]
```diff
@@ -539,3 +539,12 @@
...
 	return HS.spawn_larva(xeno_candidate, mother)
+
+/datum/game_mode/distress/proc/on_xeno_evolve(datum/source, mob/living/carbon/xenomorph/new_xeno)
+	if(SSticker.round_start_time + SHUTTLE_HIJACK_LOCK > world.time)
+		if(XENO_TIER_ONE)
+			new_xeno.upgrade_xeno(XENO_UPGRADE_ONE)
+		if(XENO_TIER_TWO)
+			new_xeno.upgrade_xeno(XENO_UPGRADE_ONE)
+		if(XENO_TIER_THREE)
+			new_xeno.upgrade_xeno(XENO_UPGRADE_ONE)
```
Comment (spookydonut) [inline code/datums/gamemodes/distress.dm]:
```suggestion
			new_xeno.upgrade_xeno(XENO_UPGRADE_ONE)

```

File: code/__DEFINES/mode.dm [hunk 2]
```diff
@@ -85,7 +85,7 @@
 				list(/obj/item/weapon/gun/smg/mp7, /obj/item/ammo_magazine/smg/mp7),\
 				list(/obj/item/weapon/gun/shotgun/double/sawn, /obj/item/ammo_magazine/shotgun/flechette),\
 				list(/obj/item/weapon/gun/smg/uzi, /obj/item/ammo_magazine/smg/uzi),\
-				list(/obj/item/weapon/gun/smg/mp5, /obj/item/ammo_magazine/smg/mp5),\
+				wlist(/obj/item/weapon/gun/smg/mp5, /obj/item/ammo_magazine/smg/mp5),\
```
Comment (spookydonut) [inline code/__DEFINES/mode.dm]:
```suggestion
				list(/obj/item/weapon/gun/smg/mp5, /obj/item/ammo_magazine/smg/mp5),\
```

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 3]
```diff
@@ -520,9 +512,7 @@
 	vendor_role = SQUAD_SPECIALIST
 	gives_webbing = FALSE
 
-	listed_products = list(
-							list("STANDARD EQUIPMENT (take all)", 0, null, null, null),
-							list("Standard Kit", 0, /obj/effect/essentials_set/basic_specialist, MARINE_CAN_BUY_UNIFORM, "white"),
+	listed_products = list(,
```
Comment (spookydonut) [inline code/game/objects/machinery/vending/new_marine_vendors.dm]:
```suggestion
	listed_products = list(
```

File: code/datums/gamemodes/distress.dm [hunk 4]
```diff
@@ -539,3 +539,12 @@
 /datum/game_mode/distress/spawn_larva(mob/xeno_candidate, mob/living/carbon/xenomorph/mother)
 	var/datum/hive_status/normal/HS = GLOB.hive_datums[XENO_HIVE_NORMAL]
 	return HS.spawn_larva(xeno_candidate, mother)
+
+/datum/game_mode/distress/proc/on_xeno_evolve(datum/source, mob/living/carbon/xenomorph/new_xeno)
+	if(SSticker.round_start_time + SHUTTLE_HIJACK_LOCK > world.time)
+		if(XENO_TIER_ONE)
```
Comment (Rohesie) [inline code/datums/gamemodes/distress.dm]: There's a `switch()` lacking here.

Comment (Rohesie) [reply code/datums/gamemodes/distress.dm]: Please don't resolve issues on your own. This one is not properly resolved.
Look for an example where you took this code from, of the correct `switch()` value to use here.

**Comment by jroinc** [comment]
-1. Just encourages more unga-rushes and 30 minute rounds. I guess we can see how it goes, but I think the 20 min dropship timer should be pretty much the minimum, as I believe this'll lead to 4-5 ungas and specs consistently launching at the minimum time, hitting the shutters and unga-ing out.

**Comment by Rohesie** [comment]
I'm not entirely convinced by this.
Better get feedback from the playerbase.

**Comment by Eskjjlj** [comment]
Xenos need marines to take a while to come so that they have enough time to build their hive.

**Comment by BurgerLUA** [comment]
This needs to be merged ASAP. The xeno being mature issue can be fixed with another PR.

**Comment by Rohesie** [comment]
This is conflicting and cannot be tested nor merged.

**Comment by Rohesie** [comment]
Closing due to staleness.
Try making smaller, atomized, changes in the future to see them tested.

**Pull request closed without merging**